### ISSUE TITLE
[MAS1.4.11] [Visual Requirements-Debug] Luminosity Ratio is less than 3:1 for “Previous and next” arrow buttons 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1815]
**Actual Result:**
Luminosity Ratio is less than 3:1 for “Previous and next” arrow buttons present in the debug mode screen 

**Expected Result:**
Luminosity Ratio should be greater than or equal to 3:1 for “Previous and next” arrow buttons 

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened
3. Bot screen upon connecting via azure/teams
4.Debug screen gets appeared on emulator 
5. Navigate to the next and previous arrow buttons 
6. Open 'Colour Contrast Analyser' application.
7. Observe the issue

**User Impact:**
Users with low vision or colour blindness needs that difference between contrast of button text  is always high.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator
Tool: Colour contrast Analyzer

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B6CA3BF14-2635-434A-832C-28DD9A18B9FC%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true&cid=c3837dea-3bb1-4c5a-9374-19ec384f14c6
![MAS1 4 11_JSON](https://user-images.githubusercontent.com/53518994/64162359-c6da0200-ce5c-11e9-9f79-1cd44f53ace0.png)

**Comment by cwhitten** [comment]
@DesignPolice can you give @denscollo an updated color value for these icons? Thanks.

**Comment by DesignPolice** [comment]
Will do, not sure where some of these colors came from in the first place. Lets just move this to Azure Blue - #0078D4 it will make things easy without adding problems. @cwhitten @denscollo

**Comment by denscollo** [comment]
Ok, will update it to use that color value. Thanks! @DesignPolice @cwhitten
